### PR TITLE
added comments

### PR DESCRIPTION
**Pull request by overcookedlobster**
created readme.md as well as comments on each file

### CODE REVIEW

**Review by AgapeDsky — APPROVED** [review]
Nambah comment doang sama dirapiin kan? Nice

**Comment by zeroleaked** [comment]
resolve conflict dulu mo, commit terakhir aku nge-flip pin i-o nya sda dan scl supaya sama dengan sky. Intinya supaya gak kebolak balik. Di interface, _o dan _i itu  aku buat jadi persepektif tb ( _o: tb outputs, _i: tb inputs)

File: sim/sim_axil/.jukit/.jukit_info.json
Comment (zeroleaked) [inline sim/sim_axil/.jukit/.jukit_info.json:1]: untrack this file

File: sim/sim_axil/i2c_agent/.jukit/.jukit_info.json
Comment (zeroleaked) [inline sim/sim_axil/i2c_agent/.jukit/.jukit_info.json:1]: untrack this file. You can add json files to gitignore.

**Pull request merged**